supplementary_style_guide/glossary_terms_conventions/general_conventions/o.adoc

```diff
@@ -479,9 +479,9 @@ Use Organization Administrator as a proper noun when referring to the Organizati
 
 *Use it*: yes
 
-*Incorrect forms*: xref:ceph-osd[ceph-osd], xref:object-storage-device[Object Storage Device], xref:osd-daemon[OSD daemon]
+*Incorrect forms*: ceph-osd, Object Storage Device, OSD daemon
```

Comment:
I know I'm late to the party here, but I'm thinking that these items were supposed to be in the "See also" line, and not in the incorrect forms section at all.

If you look at these other places this entry links to, that's how they all look. I'm thinking we should remove them here in a new PR. What do others think?

Reply:
Sure - but the "incorrect forms" line is still correct here right?

Reply:
I think it needs to be empty. I think these xrefs were mistakenly put in the "Incorrect forms" field instead of the "See also" field.

Reply:
@ndeevy any idea about the above?